manager/notifier.go

```diff
@@ -370,17 +390,22 @@ func newFlowdockMessage(op notificationOp, config *pb.FlowdockConfig, a *Alert)
 	case notificationOpResolve:
 		status = "resolved"
 	}
+	contentBuf := new(bytes.Buffer)
```

Comment:
Style nit: we only use `new` or `make` when required. Instead use `contentBuf := &bytes.Buffer{}` here (or `var contentBuf bytes.Buffer` and then `&contentBuf` as the argument).